### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by travissarles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,11 +22,15 @@ want anything else installed system-wide.
...
-
-Start the server in the context of the bundle:
-
-    bundle exec jekyll serve
+Install ruby and rbenv. `cd` into the directory where you cloned this repository, then execute the following:
+```
+rbenv install 2.4.0
+rbenv local 2.4.0
+rbenv rehash
+gem install bundle
```
Comment (ashawley) [inline README.md]: Does the command `rbenv shell 2.4.0` do the same thing?  I never knew it existed until recently.  Works well, for me.

**Comment by ashawley** [comment]
Ruby 2.4.0 doesn't work for me, or is this in advance of the reworked version of the site?

**Comment by heathermiller** [comment]
Turns out that we use Ruby 2.3.1 on the server. There's apparently no harm in keeping on version 2.3.1 – Jekyll doesn't require 2.4.0 or anything. Fabien asked that we not upgrade, yet.

That means changing deps in #606 not to require Ruby 2.4.0. I can do that. That means that these instructions would then be slightly inaccurate.

**Comment by heathermiller** [comment]
@travissarles tweaked the README, assuming #606 will be updated

**Pull request merged**